lnbits/extensions/satspay/views.py

```diff
@@ -40,6 +40,8 @@ async def display(request: Request, charge_id: str):
             "request": request,
             "charge_data": charge.dict(),
             "wallet_inkey": inkey,
-            "mempool_endpoint": onchainwallet_config.mempool_endpoint,
+            "mempool_endpoint": onchainwallet_config.mempool_endpoint
+            if onchainwallet_config
```

Comment:
imho that if kinda looks confusing, i would heck for that above and use a variable, just for clarity.

Reply:
close to happy, :D black is not :)